pkg/cmd/bootstrap.go

```diff
@@ -154,11 +159,11 @@ func missingFlagErr(flags []string) error {
 }
 
 // initiateInteractiveMode starts the interactive mode impplementation if no flags are passed.
-func initiateInteractiveMode(io *BootstrapParameters) error {
+func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client) error {
 	log.Progressf("\nStarting interactive prompt\n")
 	// ask for sealed secrets only when default is absent
-	if io.SealedSecretsService == (types.NamespacedName{}) {
-		io.SealedSecretsService.Name = ui.EnterSealedSecretService(&io.SealedSecretsService)
+	if client.CheckIfSealedSecretsExists(defaultSealedSecretsServiceName) != nil {
```

Comment:
You can get error for a number of reasons.   Service does not exist is only one of the many possibly reasons.

Reply:
Any other error will be taken care of before the interactive mode in the validateSealedSecrets()